### ISSUE TITLE
worker-php.js uses eval which breaks webpack 4.0 builds

### ISSUE THREAD

**Issue by joemasilotti** [ajaxorg/ace#4068]
I am using Ace editor with webpack but running into an issue. I recently upgraded to webpack 4.0 and am getting the following error message:

```
Uncaught (in promise) Error: Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: [directory]/node_modules/ace-builds/src-noconflict/worker-php.js: Binding 'eval' in strict mode (2397:44)

  2395 | 
  2396 | 
> 2397 | PHP.Parser = function ( preprocessedTokens, eval ) {
       |                                             ^
  2398 | 
  2399 |     var yybase = this.yybase,
  2400 |     yydefault = this.yydefault,
    at Parser.raise ([directory]/node_modules/@babel/parser/lib/index.js:6387:17)
```

Is there anything I can do about this via webpack configuration? Or do code changes need to be made to Ace itself? Thanks in advance!

**Comment by joemasilotti** [comment]
Changing `eval` to `evaluate` on `node_modules/ace-builds/src-noconflict/worker-php.js:2397` and `2519` fixes the issue. However, I don't see those files in the source. Are they generated? If so, is there a way to change that parameter name via configuration?

**Comment by joemasilotti** [comment]
This is fixed in #4070.
